### ISSUE TITLE
daemon, cli: prepend timestamp for every log line

### ISSUE THREAD

**Issue by FeelyChau** [alibaba/pipcook#533]
Now, the log from pipcook-cli is pure text without log time, it's not good to analyze problems.
If it could look like this, that would be great:
```shell
[18:12:23] ⚠ Building prefix dict from the default dictionary ...
[18:12:24] ⚠ Loading model from cache /var/folders/tq/cmmgdq_143jbd5w_drrdl8km0000gp/T/jieba.cache
[18:12:24] ⚠ Loading model cost 0.703 seconds.
[18:12:25] ⚠ Prefix dict has been built successfully.
[18:12:27] ℹ create a result "qy5nvaid" for plugin "@pipcook/plugins-bayesian-model-evaluate@1.1.0"
[18:12:30] ℹ [job] running modelEvaluate end
[18:13:00] ℹ [job] run successfully
[18:13:02] ✔ job is finished successfully
[18:13:04] ℹ start to download output to /Users/feiyu.zfy/work/pipcook/output
[18:13:10] ✔ download finished
```
We could [extend the event interface](https://github.com/alibaba/pipcook/blob/add/job-step-event/packages/daemon/src/service/trace-manager.ts#L17) to put the time stamp, after #526 , it's easy to implement, then we could add an option to the pipcook-cli command like `pipcook run pipeline.json -t` to show it (or default).

**Comment by yorkie** [comment]
How do you prepend the timestamp?

**Comment by yorkie** [comment]
I have opened a PR https://github.com/sindresorhus/ora/pull/154, which adds a function option `prefixText` to prepend a custom string before every line.

> BTW, updated the title, and please write your issue title to be more patient and follow the rule that I mentioned at #532   @FeelyChau.

**Comment by FeelyChau** [comment]
> I have opened a PR [sindresorhus/ora#154](https://github.com/sindresorhus/ora/pull/154), which adds a function option `prefixText` to prepend a custom string before every line.
> 
> > BTW, updated the title, and please write your issue title to be more patient and follow the rule that I mentioned at #532   @FeelyChau.

It's not what I needed, the time stamp should be stored in the disk, with log records and events, so when we call the `job.log`, or `plugin.installLog`, the information of time is still here.

**Comment by yorkie** [comment]
https://github.com/sindresorhus/ora/pull/154 has been merged and ora@5.1.0 is released.

**Comment by FeelyChau** [comment]
Done by #848 .
